packages/apollo-link-state/src/__tests__/client.ts

```diff
@@ -139,10 +145,12 @@ describe('cache usage', () => {
     `;
 
     const local = withClientState({
-      Mutation: {
-        start: (_, $, { cache }: { cache: InMemoryCache }) => {
-          cache.writeQuery({ query, data: { field: 1 } });
-          return { start: true };
+      resolvers: {
+        Mutation: {
+          start: (_, $, { cache }: { cache: InMemoryCache }) => {
```

Comment:
Lol, this arguments list is hilarious - I wish JS had a way to skip args.

Reply:
One option below - but you can't use arrow functions.. I think the _ usage that is popular now messes with my usage of lodash/_ if on the same file. That's my only complaint! 

```js
const skipArgs = function() {
  const info = arguments[3]
  return info
}
const result = skipArgs(1, 2, 3, 'important info')
console.log('result?', result) // important info
```